wifi/scheme.py

```diff
@@ -158,6 +158,12 @@ def activate(self):
 
         return self.parse_ifup_output(ifup_output)
 
+    def deactivate(self):
+        """
+        Disconnect from the network without bringing down the interface
+        """
+        return subprocess.call(['/sbin/dhclient', '-r', self.interface], stderr=subprocess.STDOUT)
```

Comment:
what does `subprocess.call` actually return? I don't think we need to return anything from here.